### PR TITLE
DevTools / console enhancements

### PR DESCRIPTION
**Pull request by lbogdan**
See commit messages.

### CODE REVIEW

File: src/sandbox/console/index.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
 import { dispatch } from 'codesandbox-api';
 
-import massageJSON from './massage-json';
+import messageJSON from './message-json';
```
Comment (CompuIves) [inline src/sandbox/console/index.js]: Haha, this wasn't a typo, but a bit cryptic. Maybe rename it to transform-json?

File: src/app/components/sandbox/Preview/DevTools/index.js [hunk 2]
```diff
@@ -230,7 +232,11 @@ export default class DevTools extends React.PureComponent<Props, State> {
         // happen when pointer events are disabled and in turn disables scroll.
         // It's hacky, but it's to fix a bug in the browser.
         setTimeout(() => {
-          this.setState({ height: this.state.height + 1 });
+          const height = this.state.height + 1;
+          this.setState({ height });
+          if (height > 64) {
+            store.set('devtools.height', height);
```
Comment (CompuIves) [inline src/app/components/sandbox/Preview/DevTools/index.js]: Is this to inspect the height of the devtools?

File: src/app/utils/debug.js [hunk 3]
```diff
@@ -15,6 +17,12 @@ const getDebugger = () => {
     };
   }
 
+  if (store.get('debug.disabled')) {
```
Comment (CompuIves) [inline src/app/utils/debug.js]: Hmm, maybe we can solve this differently. Debug has the functionality to check based on env var DEBUG=<prefix> what messages to show. Maybe we can put this in localStorage and derive it from that.

File: src/app/components/sandbox/Preview/DevTools/index.js (hunk 2 above)
Comment (CompuIves) [reply src/app/components/sandbox/Preview/DevTools/index.js]: Nevermind, read the commit. Reaaaaally cool!

File: src/sandbox/console/index.js (hunk 1 above)
Comment (lbogdan) [reply src/sandbox/console/index.js]: Haha, **now** I get it!

File: src/app/utils/debug.js (hunk 3 above)
Comment (lbogdan) [reply src/app/utils/debug.js]: So is it possible to disable debug messages by running `DEBUG=*something* yarn start`? And if yes, what should `*something*` be?

File: src/sandbox/console/index.js (hunk 1 above)
Comment (lbogdan) [reply src/sandbox/console/index.js]: Renamed it to `transform-json.js` / `transformJSON`.

File: src/app/utils/debug.js (hunk 3 above)
Comment (lbogdan) [reply src/app/utils/debug.js]: (removed it for now)

Comment (CompuIves) [reply src/app/utils/debug.js]: We do by default DEBUG=*, but if you omit DEBUG you see no messages. You can for example do DEBUG=cs:compiler:* to see all compiler debug messages.

File: src/sandbox/console/index.js (hunk 1 above)
Comment (CompuIves) [reply src/sandbox/console/index.js]: Great! Yeah massage sounds a bit strange. Heard it last week at the conference haha

File: src/app/utils/debug.js (hunk 3 above)
Comment (lbogdan) [reply src/app/utils/debug.js]: That's strange, because I can see all `cs:` messages by default. Maybe because I'm on Windows? :thinking:

Comment (CompuIves) [reply src/app/utils/debug.js]: I enable it by default 😅  (https://github.com/CompuIves/codesandbox-client/blob/master/src/app/utils/debug.js#L19-L20). We can derive this from the localStorage maybe.

Comment (lbogdan) [reply src/app/utils/debug.js]: I commented out that line, and it still showed all messages. That's why I conditionally (based on localStorage) did `debug.disable()` in the first place.

Comment (CompuIves) [reply src/app/utils/debug.js]: Hmm, okay that's strange. Then their logic changed to enable by default.

**Comment by CompuIves** [comment]
Ahhh I created a merge conflict. Shouldn't be too hard to fix.

File: src/app/utils/debug.js (hunk 3 above)
Comment (lbogdan) [reply src/app/utils/debug.js]: I'll just disable it manually for now. (and be careful to not commit it! :laughing: )

**Pull request merged**